atlas-fastq-provider-functions.sh

```diff
@@ -963,6 +997,104 @@ fetch_file_from_ena_over_ssh() {
     return 0
 }
 
+fetch_file_from_ena_over_s3() {
+    ##########
+    local enaFile=$1
+    local destFile=$2
+    local retries=${3:-3}
+    local library=${4:-''}
+    local validateOnly=${5:-''}
+    local downloadType=${6:-'fastq'}
+    local status=${7:-'public'}
+    local tempdir=$(get_temp_dir)
+
+    check_variables "enaFile" "destFile"
+
+    check_ena_s3
```

Comment:
```suggestion
    check_ena_s3_profile
```

Reply:
Done in 27c8b346819a046c361956a4e6c96d5d7a70094d